src/System.ComponentModel.TypeConverter/src/Resources/System.ComponentModel.TypeConverter.rd.xml

```diff
@@ -13,6 +13,9 @@
         </Type>
         <Type Name="TypeConverterAttribute">
           <AttributeImplies Activate="Required Public" />
+          
+          <!-- AttributeCollection.GetDefaultAttribute will end up reflecting on the Default field -->
+          <Field Name="Default" Dynamic="Required" />
```

Comment:
This makes this attribute self-rooting. Does it need to be rooted?

Reply:
The thing I would want to go for is "if the owning type is getting any sort of metadata, also root and generate metadata for this field", but I don't believe we have means to express that in RD.XML.

Rooting this field is pretty harmless.